dev/io.openliberty.jakarta.rest.4.0.internal_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,618 @@
+            <artifactId>arquillian-junit5-container</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!-- see: https://github.com/arquillian/arquillian-core/issues/248 -->
+        <!-- and https://github.com/arquillian/arquillian-core/pull/246/files -->
+        <dependency>
+            <groupId>org.jboss.arquillian.protocol</groupId>
+            <artifactId>arquillian-protocol-servlet-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <!--        <dependency>
+                    <groupId>org.junit.jupiter</groupId>
+                    <artifactId>junit-jupiter-params</artifactId>
+                    <scope>test</scope>
+                </dependency>-->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <finalName>${project.artifactId}</finalName>
+        <directory>${targetDirectory}</directory>
+        <defaultGoal>clean build-classpath test integration-test</defaultGoal>
+        <testResources>
+            <testResource>
+                <directory>src/main/resources</directory>
+                <includes>
+                    <include>*</include>
+                </includes>
+            </testResource>
+        </testResources>
+        <plugins>
+<!-- The following were added in an attempt to enable the JAXRSSigTestIT test that is currently excluded.   Leaving this in case we re-visited attempting to enable this test 
+     that requires the addition of properties to the signature.sigTestClasspath below.
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.9</version>
+                
+                <executions>
+                    <execution>
+     
+                        <id>build-classpath</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>build-classpath</goal>
+                        </goals>
+                        <configuration>
+                            
+			              <target>
+			                <copy file="${io.openliberty.jakarta.restfulWS.3.1}" tofile="${project.build.directory}/signaturedirectory/jakarta.ws.rs-api.jar"/>
```

Comment:
This would be 4.0 of course instead of 3.1

Reply:
This is a commented block that may be removed soon, but changed anyway.